### DISCUSSION TITLE
The Stretchy Flexbox

### DISCUSSION THREAD

**Discussion by meviru** [meviru/badge-hunting#13] [Galaxy Trivia] [answered]
You have a container that wants to practice yoga and stretch to fill all available space. Which CSS property gives it the flexibility it craves?

**Comment by DarmorGamz** [comment, marked as answer]
You can use the ```flex-grow``` property to really show off your flexibility and fill all available space.

**Reply by meviru** [reply]
That is correct!! @DarmorGamz 👏👏
Answer one more question and get a Galaxy Brain Badge on your profile.
